<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pacientes_view</class>
 <widget class="QMainWindow" name="pacientes_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color:gray;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="documentMode">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="icon">
           <iconset>
            <normaloff>ELEMENTOS GRAFICOS/silla-de-dentista.png</normaloff>ELEMENTOS GRAFICOS/silla-de-dentista.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Pacientes registrados</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame{
background-color:rgb(70, 130, 169);
	font: 12pt &quot;Microsoft Sans Serif&quot;;
}
QTableWidget{
color:rgb(0,0,0);
gridline-color:rgb(0,206,151);
font-size:12pt;
}
QHeaderView::section{
background-color:rgb(0,206,151);
border:1px solid rgb(0,0,0);
font-size:12pt;
}
QTbaleWidget QTableCornerButton::section{
background-color:rgb(0,0,0);
border:1px solid  rgb(0,206,151);
}
QLineEdit{
border:2px solid rgb(0,0,0);
border-radius:20px;
	font: 12pt &quot;Microsoft Sans Serif&quot;;
	background-color: rgb(255, 255, 255);
}
QPushButton{
border-radius:20px;
border:2px solid rgb(0,0,0);
background-color:white;
}
QPushButton:hover{
	background-color:#CB4335;
}
QTextEdit{
border:4px solid rgb(0,0,0);
border-radius:20px;
	font: 12pt &quot;Microsoft Sans Serif&quot;;
	background-color: rgb(255, 255, 255);
}
QLabel{
	
	font: 12pt &quot;Microsoft Sans Serif&quot;;
}
</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Visualizaci√≥n de pacientes</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QComboBox" name="filtro">
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>300</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="frame">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="in_buscar">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>200</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="inputMask">
                   <string/>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="maxLength">
                   <number>20</number>
                  </property>
                  <property name="placeholderText">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="bt_buscar">
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>ELEMENTOS GRAFICOS/buscar.png</normaloff>ELEMENTOS GRAFICOS/buscar.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="bt_act">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>150</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>9</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Actualizar tabla</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>ELEMENTOS GRAFICOS/actualizar.png</normaloff>ELEMENTOS GRAFICOS/actualizar.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="bt_preview">
                  <property name="minimumSize">
                   <size>
                    <width>150</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>150</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>9</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Guardar PDF</string>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QFrame" name="frame_3">
                <property name="styleSheet">
                 <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QTableWidget" name="tabla_p">
                   <attribute name="horizontalHeaderMinimumSectionSize">
                    <number>100</number>
                   </attribute>
                   <attribute name="horizontalHeaderDefaultSectionSize">
                    <number>170</number>
                   </attribute>
                   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="horizontalHeaderStretchLastSection">
                    <bool>true</bool>
                   </attribute>
                   <attribute name="verticalHeaderVisible">
                    <bool>true</bool>
                   </attribute>
                   <column>
                    <property name="text">
                     <string>ID</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Nombre de usuario</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Cedula</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Nombres</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Apellidos</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Edad</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Sexo</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Telefono</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Fecha_Diagnotico</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionVolver_al_menu_principal"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionVolver_al_registro">
   <property name="text">
    <string>Volver al registro</string>
   </property>
  </action>
  <action name="actionVolver_al_menu_principal">
   <property name="text">
    <string>Volver al menu principal</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
